test/tst_loss.jl

```diff
@@ -24,6 +52,8 @@ function test_value(l::SupervisedLoss, f::Function, y_vec, t_vec)
     @testset "$(l): " begin
         for y in y_vec, t in t_vec
             @test abs(value(l, y, t) - f(y, t)) < 1e-10
+            # TODO: consider this test?
+            # @test abs(value_deriv(l,y,t)[1] - f(y, t)) < 1e-10
```

Comment:
this is covered by other tests. see: https://github.com/JuliaML/Losses.jl/blob/master/test/tst_loss.jl#L74